### ISSUE TITLE
Not training in DIGITS

### ISSUE THREAD

**Issue by achaiah** [jay-mahadeokar/pynetbuilder#6]
Hi, I was wondering if you tried running the models in NVIDIA DIGITS?  I imported one of your models (resnet 36) - it runs but does not train.  Any ideas?

Thanks!

**Comment by jay-mahadeokar** [comment]
I haven't tried it. What do you mean by does not train? Can you be more specific?

**Comment by achaiah** [comment]
It literally doesn't train (i.e. zero progress over 200 epochs). I found a fix here: https://github.com/BVLC/caffe/pull/3919  There's a difference in BatchNorm between bvlc and NVIDIA that is confusing.

**Comment by mrgloom** [comment]
@achaiah  Can you be more specific about differences in BN layer?
Seems realted thread is here ? https://github.com/NVIDIA/DIGITS/issues/629

**Comment by achaiah** [comment]
The thread you pointed to is correct as well... same issue. The CUDNN BN implementation has changed and that has broken existing nets that use BN.
